package.json

```diff
@@ -11,8 +11,12 @@
     "generate": "nuxt generate"
   },
   "dependencies": {
+    "@nuxtjs/axios": "^5.9.2",
+    "animate.css": "^3.7.2",
     "nuxt": "^2.0.0",
-    "@nuxtjs/axios": "^5.3.6"
+    "socket.io": "^2.3.0"
```

Comment:
Aangezien die sockets pagina eruit is, waar is deze voor nodig?? gaan we die later alsnog gebruiken?